support-frontend/cloud-formation/cfn.yaml

```diff
@@ -301,7 +302,8 @@ Resources:
     Properties:
       AlarmActions:
       - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
-      AlarmName: !Sub High 5XX rate for support-frontend in ${Stage}
+      AlarmName: !Sub HIGH severity - support-frontend instances are returning 5XX errors in ${Stage}
+      AlarmDescription: Impact - Some or all actions on support website are failing. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit
```

Comment:
Is this alarm strictly necessary? I don't remember seeing high 5XX from target and that not also be reflected by the ELB having high 5XX

Reply:
yes it is needed.  The reports are Either/OR, rather than one including the other.